specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -740,6 +798,32 @@
           "type": "string"
         }
       }
+    },
+    "AcsUserEngagement": {
```

Comment:
You don't need a separate definition for this. The x-ms-enum can be applied inline in the property.